pkg/cmd/cli/backup/create.go

```diff
@@ -26,6 +26,8 @@ import (
 	"k8s.io/client-go/tools/cache"
 
 	api "github.com/heptio/velero/pkg/apis/velero/v1"
+	velerov1api "github.com/heptio/velero/pkg/apis/velero/v1"
```

Comment:
just noticed we've got a duplicate import here with the previous line - let's convert all `api` uses to `velerov1api`.

Reply:
oops, good spot